### ISSUE TITLE
Idea!!

### ISSUE THREAD

**Issue by Jaspenton** [unixpickle/kahoot-hack#94]
Can u please make it so we can choose which answer the answer please

**Comment by unixpickle** [comment]
kahoot-play lets you do that.

**Comment by Jaspenton** [comment]
Does that let you choose nick and number of bots?

**Comment by unixpickle** [comment]
Nope. Are you suggesting that, for example, kahoot-flood should let you choose answers? That would theoretically be possible.

**Comment by Jaspenton** [comment]
Yes that's what I ment

**Comment by Noxturnix** [comment]
Allow users to enter the answer when the question is appeared. ;)
